source/transactions/tests/retryable-commit.yml

```diff
@@ -218,7 +218,7 @@ tests:
         object: session0
         result:
           errorCodeName: Interrupted
-          errorLabelsOmit: ["TransientTransactionError"]
+          errorLabelsOmit: ["RetryableWriteError", "TransientTransactionError"]
```

Comment:
Does it make sense to add UnknownTransactionCommitResult to the list of omitted labels? That may be similar to how you have RetryableWriteError and TransientTransactionError included in assertions for the UnknownTransactionCommitResult tests in `error-labels.yml`.

Reply:
Yeah I think that makes sense! I added it in this test and one other.